StantDrawing/BrushTool/GroupSizeView.swift

```diff
@@ -0,0 +1,287 @@
+        
+        setButtonsAsNotClicked(buttons: groupButtons)
+        
+        // Set the second SizeButton as Default
+        setButtonSizeAsClicked(button: groupButtons[1].uiButton)
+        
+        return groupButtonsView
+    }
+    
+    private func setButtonsAsNotClicked(buttons: [Button]) {
+        buttons.forEach { button in
+            setButtonSizeAsNotClicked(button: button.uiButton)
+        }
+    }
+    private func setButtonSizeAsClicked(button: UIButton) {
+        let darkGray = UIColor(hex: "606060")
+        button.alpha = 1
+        button.layer.cornerRadius = button.frame.height / 2
+        button.backgroundColor = darkGray
+    }
+    
+    private func setButtonSizeAsNotClicked(button: UIButton) {
+        let darkGray = UIColor(hex: "606060")
+        button.alpha = 0.4
+        button.layer.cornerRadius = button.frame.height / 2
+        button.backgroundColor = darkGray
+    }
+    
+    
+    private func createReferenceIconSizeView(iconReferenceName: String) -> UIView {
+        let iconView = UIView()
+        
+        let originIconView = CGPoint(x: 0, y: 0)
+        let sizeIconView   = CGSize(width: groupViewWidth, height: 60)
+        let frameIconView  = CGRect.init(origin: originIconView, size: sizeIconView)
+        
+        iconView.frame              = frameIconView
+        iconView.backgroundColor    = backgroundGroupGrayColor
+        iconView.layer.cornerRadius = iconView.frame.width / 2
+        
+        let originIconImage = CGPoint(x: (groupViewWidth - 25) / 2, y: 18)
+        let sizeIconImage   = CGSize(width: 25, height: 25)
+        let frameIconImage  = CGRect.init(origin: originIconImage, size: sizeIconImage)
+        
+        let iconImageView = UIImageView(image: UIImage(named: iconReferenceName, in: bundle, compatibleWith: nil))
+        
+        let identifierName = "\(iconReferenceName)SizeReferenceIdentifier"
+        setIdentifierForView(view: iconImageView, identifierName: identifierName)
+        iconImageView.frame                   = frameIconImage
+        
+        iconView.addSubview(iconImageView)
+        
+        return iconView
+    }
+    
+    private func createSeparatorView(yReference: CGFloat) -> UIView {
+        let separatorView   = UIView()
+        let originSeparator = CGPoint(x: 10, y: yReference)
+        let sizeSeparator   = CGSize(width: groupViewWidth - 20, height: 1)
+        let frameSeparator  = CGRect.init(origin: originSeparator, size: sizeSeparator)
+        
+        separatorView.frame           = frameSeparator
+        separatorView.backgroundColor = UIColor.black
+        separatorView.alpha           = 0.2
+        
+        return separatorView
+    }
+
+    private func configureGroupLayout(buttons: [Button], groupView: UIView, groupViewOirigin: CGPoint, addAsSubview: Bool = true) {
+        let _: [Button] = buttons.enumerated().map() { (index, button) in
```

Comment:
Why are you using a map if you are not expecting a return value?

Reply:
Changed to use a for.